core/outbox.go

```diff
@@ -70,8 +70,9 @@ func (ob *Outbox) Queue() *MessageQueue {
 }
 
 // Send marshals and sends a message, retaining it in the outbound message queue.
+// If publish is true, it publishes the message to the network at the current block height.
 func (ob *Outbox) Send(ctx context.Context, from, to address.Address, value types.AttoFIL,
-	gasPrice types.AttoFIL, gasLimit types.GasUnits, method string, params ...interface{}) (out cid.Cid, err error) {
+	gasPrice types.AttoFIL, gasLimit types.GasUnits, method string, publish bool, params ...interface{}) (out cid.Cid, err error) {
```

Comment:
I propose moving the `publish` parameter one slot earlier so that the params directly follow the method name and the bool cannot be mistaken for a parameter.